kytos/core/interface.py

```diff
@@ -494,7 +494,9 @@ def __eq__(self, other):
 
     def is_valid(self):
         """Check if TAG is possible for this interface TAG pool."""
-        if self.user_tag:
+        if self.user_tag is not None:
```

Comment:
Aldo, let's avoid nested ifs when we can afford to have flat ones or early returns. 

Could you adapt this accordingly?

Reply:
Done in commit [3b3fa3129363cb7b32ed5896aa16285a86889ba0](https://github.com/kytos-ng/kytos/pull/331/commits/3b3fa3129363cb7b32ed5896aa16285a86889ba0)